### DISCUSSION TITLE
Use Backup process wirelessly?

### DISCUSSION THREAD

**Discussion by Chancynng** [outdoorbits/little-backup-box#163] [Q&A]
I have installed LBB on my Pi Model 3B with Raspberry Pi OS Lite x64
When I have my ethernet cable plugged into the device and power it on, I'm able to use the backup feature.
Whenever I plug the device in to power without the ethernet cable plugged in it brings me to the normal Raspberry Pi command line and nothing happens when a drive gets plugged in. 
Do I have to have network access for the Pi to work?
Is there a way to make it work so All I need is power, source disk and target disk? (This instance is copying from a SD card to a Flash Drive for testing)

**Comment by ruebyi** [comment]
You can define a standard backup mode - in my Case I use device to external storage. 

I only need to plug my ssd in, wait that it’s mounted and then plug-in an SD Card. 

It’s helpful to have a display installed if you don’t use it with the browser. 

I also set it up with comitup. So if I don’t have an Ethernet connection it will open a wifi I can connect with.

**Reply by Chancynng** [reply]
I did set it up without comitup. Is this needed for it to work without being connected a network?

**Comment by ruebyi** [comment]
No, only that you get that Wi-Fi to connect directly

**Reply by Chancynng** [reply]
I don't know how else to say it.
When connected to an ethernet cable, I let the device power on and I can connect the target and then the source and it will copy the contents no problem.
If I DON'T plug in any ethernet cable and let the device power on fully when I connect target and the source nothing happens. The contents never get copied, the pi doesn't shut itself down.

**Reply by Chancynng** [reply]
I have a display connected through HDMI in both instances

**Reply by ruebyi** [reply]
Hmm, ok, I installed it headless and the display is just the small status display…

is there a standard backup mode configured in the settings?

**Reply by Chancynng** [reply]
> Hmm, ok, I installed it headless and the display is just the small status display…
> 
> is there a standard backup mode configured in the settings?

I have gone ahead and started reflashing the microSD. This time I'm installing comitup to see if it makes any difference. What's the process difference for a headless installation?

**Reply by Chancynng** [reply]
as for the standard backup mode. During installation I selected source -> ext drive

**Reply by ruebyi** [reply]
The headless installation is a raspiOs without desktop

**Reply by Chancynng** [reply]
> The headless installation is a raspiOs without desktop

I'm Raspberry Pi OS Lite 64bit for my OS. Would using the 32 bit OS change anything or make any difference for LBB

**Reply by ruebyi** [reply]
No, if youngsters a pi4 use the 64bit

**Comment by Chancynng** [comment]
The issue has been resolved. For some reason, when not installing comitup it wouldn't let me transfer files between storage mediums without having an ethernet cable plugged in. Not sure why but after reflashing the OS with comitup, the device is working completely fine wireless and not attached to a display.
